Patches/Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_CrescendoAnole.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>Alpha Animals</li>
+		</mods>
+		<match Class="PatchOperationSequence">
+			<operations>
+				<li Class="PatchOperationAddModExtension">
+					<xpath>/Defs/ThingDef[defName="AA_CrescendoAnole"]</xpath>
+					<value>
+						<li Class="CombatExtended.RacePropertiesExtensionCE">
+							<bodyShape>QuadrupedLow</bodyShape>
+						</li>
+					</value>
+				</li>
+				<li Class="PatchOperationAdd">
+					<xpath>/Defs/ThingDef[defName="AA_CrescendoAnole"]/statBases</xpath>
+					<!-- No idea for these just did random numbers. -->
+					<!-- (Note this is a lizard that grows strong when angered) -->
+					<value>
+						<MeleeDodgeChance>0.03</MeleeDodgeChance>
+						<MeleeCritChance>0.25</MeleeCritChance>
+						<MeleeParryChance>0.10</MeleeParryChance>
```

Comment:
These are quite close to statbases of a rough-plated monitor. A good starting point, to be fair. As crescendo anoles are more intelligent, a tad bit higher crit chance and unchanged dodge and parry chance (compared to the monitor, which is much larger and has greater reach and step length) is reasonable. If anything

Reply:
Done(?)